controllers/news.controllers.js

```diff
@@ -0,0 +1,65 @@
+const { findingTopics, findingArticleId, findingUsers, patchingArticleId, findingArticles, findingComments } = require('../models/news.models.js')
+
+exports.getTopics = (req, res, next) => {
+    findingTopics().then((arrayOfTopics) => {
+        res.status(200).send({ topics: arrayOfTopics })
+    })
+        .catch((err) => {
+            next(err);
+        });
+}
+
+exports.getArticles = (req, res, next) => {
+    const { topic } = req.query;
+
+    findingArticles(topic).then((result) => {
+        res.status(200).send({ result })
+    })
+        .catch((err) => {
+            next(err);
+        });
+}
+
+exports.getArticleId = (req, res, next) => {
+    const { article_id } = req.params
+
+    findingArticleId(article_id).then((article) => {
+        res.status(200).send(article)
+    })
+        .catch((err) => {
+            next(err);
+        });
+}
+
+exports.patchArticleId = (req, res, next) => {
+    patchingArticleId(req.params.article_id, req.body)
+        .then((updatedArticleId) => {
+            res.status(201).send({ updatedArticleId });
```

Comment:
`updatedArticleId` is a strange key to use - you're not just sending back an id

just "`article`" would be fine!